cog.yaml

```diff
@@ -22,19 +22,20 @@ build:
     - "py7zr"
     - "fire"
     - "datasets"
-    - "git+https://github.com/huggingface/transformers.git@d2295708a64feff485bb40caf0decf7bae5a05a2"
+    - "transformers==4.33.2"
     - "peft==0.4.0"
     - "tensorizer==1.0.1"
     - "accelerate"
     - "scipy"
     - "bitsandbytes" # force upgrade afterwards?
-    - "https://r2.drysys.workers.dev/torch/torch-2.1.0a0+git3af011b-cp311-cp311-linux_x86_64.whl"
     - "https://r2.drysys.workers.dev/nyacomp-0.0.5-cp311-cp311-linux_x86_64.whl"
     - "trl==0.5.0"
     - "aiohttp[speedups]"
+    - "https://github.com/replicate/vllm-with-loras/raw/moin/lora_weight_space/vllm-0.1.7a0-cp311-cp311-linux_x86_64.whl"
   run:
     - curl -o /usr/local/bin/pget -L "https://github.com/replicate/pget/releases/download/v0.0.3/pget#cacheburst" && chmod +x /usr/local/bin/pget
     - curl -o /usr/local/bin/remotefile -L "https://r2.drysys.workers.dev/remotefile" && chmod +x /usr/local/bin/remotefile
+    - pip3 install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cu118
```

Comment:
why do we need to install `torchvision` & `torchaudio`?

Reply:
We don't, removed!